CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 - Fix issue with categorical explorer counts
   ([#587](https://github.com/mozilla/glam/pull/587))
+- Implement Probe Search Service
```

Comment:
Implement sounds like this PR created the search service instead of updating the search to use the search service.